dal/helpers/parsers.py

```diff
@@ -156,16 +154,14 @@ def eval_config(self, _config: str, *__):
         try:
             obj = scopes.from_path(_config_name, scope="Configuration")
 
-        except KeyError as e:
-            raise Exception(f"Configuration {_config_name} does not exist") from e
+        except KeyError as exc:
+            raise Exception(f"Configuration {_config_name} does not exist") from exc
```

Comment:
Sonar error on this line, should be :
```suggestion
            raise ValueError(f"Configuration {_config_name} does not exist") from exc
```